scenarios/httpclient.benchmarks.yml

```diff
@@ -27,6 +27,12 @@ variables:
     useWinHttpHandler: false
     useHttpMessageInvoker: false
 
+    headersDictionary:
+      none: '' # supported in WRK
+      connectionclose: '--header "Connection: close"' # supported in WRK
+      expectcontinue: '--header "Expect: 100-continue"'
```

Comment:
'connectionclose' is spelled like this because of WRK https://github.com/dotnet/crank/blob/5fdefc136b45953c3d51d2bfca5930377e2ae2ce/src/Microsoft.Crank.Jobs.Wrk/wrk.yml#L7 😞 so `expectcontinue` simply followed the same format. I have other thoughts on how we can pass the headers to WRK, so if you feel that there's no immediate need in supporting 'connectionclose' in WRK I will be happy to rename that

Reply:
Yeah, let's keep it as is, not worth the hassle.